examples/linear_model/plot_sgd_early_stopping.py

```diff
@@ -56,9 +56,9 @@
 print(__doc__)
 
 
-def load_mnist(n_samples=None, class_0=0, class_1=8):
+def load_mnist(n_samples=None, class_0='0', class_1='8'):
     """Load MNIST, select two classes, shuffle and return only n_samples."""
-    mnist = fetch_mldata('MNIST original')
+    mnist = fetch_openml('mnist_784', version=1)
```

Comment:
(alternative) Maybe we can add a link to openml in the examples (e.g., https://www.openml.org/d/554) so that users can get more information about the dataset.